packages/react-pagination/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@wpmudev/react-pagination",
+  "version": "1.0.0",
+  "description": "Now I’m the model of a modern major general / The venerated Virginian veteran whose men are all / Lining up, to put me up on a pedestal / Writin’ letters to relatives / Embellishin’ my elegance and eloquence / But the elephant is in the room / The truth is in ya face when ya hear the British cannons go / BOOM",
+  "keywords": [],
+  "author": "veljkocukic <veljkocukic@gmail.com>",
+  "license": "ISC",
+  "main": "lib/react-pagination.js",
```

Comment:
This is something that's not included in the documentation but you need to modify this section. Remove this line and write instead the following:
```
"main": "dist/react-pagination.cjs.js",
"module": "dist/react-pagination.esm.js",
"src": "lib/react-pagination.js",
```